src/Context/Context.js

```diff
@@ -1,4 +1,5 @@
 import React, { useEffect, useState } from "react";
+
 export const ContextData = React.createContext();
```

Comment:
Remove empty Lines.
Dont create extra things which are useless, it makes you confused, and others too